features/admin.claims.v1/components/edit/local-claim/edit-additional-properties-local-claims.tsx

```diff
@@ -60,6 +65,8 @@ export const EditAdditionalPropertiesLocalClaims:
 
         const [ isSubmitting, setIsSubmitting ] = useState<boolean>(false);
 
+        const RESTRICTED_PROPERTY_KEYS: string[] = [ "isUnique" ];
```

Comment:
Move this to the claim-management-constants

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7106/commits/3088d327c477257341387a57516db4d9929fbfaa